core-plugins/src/main/java/io/cdap/plugin/transform/LogParserTransform.java

```diff
@@ -216,12 +228,14 @@ private String getLog(StructuredRecord input) {
     }
   }
 
-  private void validateInputSchemaType(Schema.Type inputSchemaType) {
+  private void validateInputSchemaType(Schema inputSchema, String inputName, @Nonnull FailureCollector collector) {
```

Comment:
nit: we only tag nullable fields. So can remove @Nonnull check

Reply:
Fixed